cpp/src/transform/encode.cu

```diff
@@ -76,7 +76,7 @@ std::pair<std::unique_ptr<table>, std::unique_ptr<column>> encode(
 
     keys_table = cudf::detail::gather(keys_table->view(),
                                       gather_map_column,
-                                      cudf::detail::out_of_bounds_policy::FAIL,
+                                      cudf::out_of_bounds_policy::NULLIFY,
```

Comment:
This too.

Reply:
`FAIL` policy is no longer supported, I believe I checked this with Ashwin that the replacement is fine.

Could you confirm? @shwina

Reply:
I think this should be `IGNORE` then.

Reply:
Done.